editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import Card from "../shared/Preview/Card";
+import CardHeader from "../shared/Preview/CardHeader";
+import { PublicProps } from "../ui";
+import { MapAndLabel } from "./model";
+
+type Props = PublicProps<MapAndLabel>;
+
+function MapAndLabelComponent(props: Props) {
+  return (
+    <Card handleSubmit={props.handleSubmit} isValid>
+      <CardHeader
+        info={props.info}
+        policyRef={props.policyRef}
+        howMeasured={props.howMeasured}
+      />
```

Comment:
nit: similar to other comments, let's get `title` & `description` props in here to start too !